core/assets/bundles/bundle.properties

```diff
@@ -375,9 +375,9 @@ zone.overgrowth.description = This area is overgrown, closer to the source of th
 zone.tarFields.description = The outskirts of an oil production zone, between the mountains and desert. One of the few areas with usable tar reserves.\nAlthough abandoned, this area has some dangerous enemy forces nearby. Do not underestimate them.\n\n[lightgray]Research oil processing technology if possible.
 zone.desolateRift.description = An extremely dangerous zone. Plentiful resources, but little space. High risk of destruction. Leave as soon as possible. Do not be fooled by the long spacing between enemy attacks.
 zone.nuclearComplex.description = A former facility for the production and processing of thorium, reduced to ruins.\n[lightgray]Research the thorium and its many uses.\n\nThe enemy is present here in great numbers, constantly scouting for attackers.
-zone.fungalPass.description = A transition area between high mountains and lower, spore-ridden lands. A small enemy reconnaissance base is located here.\nDestroy it.\nUse Dagger and Crawler units. Take out the two cores.
+zone.fungalPass.description = A transition area between high mountains and lower, spore-ridden lands. A small enemy reconnaissance base is located here.\nDestroy it.\nUse Dagger and Crawler units. Take out the two cores. 
```

Comment:
TBH it takes away from the element of surprise of the game when ways to defeat the enemy are stated. That would probably count as handholding.

Remember that you are a player discovering a new zone, and imagine you are "flying in". The only things you know obviously and initially are the environment. The zone shows you its environment, but you have to figure out how to defeat what's in it, just like how you see the environment of the board game you play, and you know the rules, but it doesn't give you every tip on defeating your opponent.

Reply:
@Luxray5474 i think it was anuken who did the original ones
i think it would be good if the hints were removed and we just kept the description